CHANGELOG

```diff
@@ -1,7 +1,9 @@
 --- CHANGELOG ---
---- Assimulo-3.4.4 ---
+--- Assimulo-3.5.0 ---
```

Comment:
Should we publish this under version 4.0.0 instead considering it is a breaking change?

Reply:
Assuming we (somewhat) follow semantic versioning, it is not breaking. The public API did not change and there are no changes to the results. 

The dependencies contain breaking changes. However, this problem is to be solved by anyone including Assimulo as a dependency, since the question whether this is an actual problem depends on the other dependencies within that project.

See also [https://semver.org/#what-should-i-do-if-i-update-my-own-dependencies-without-changing-the-public-api](https://semver.org/#what-should-i-do-if-i-update-my-own-dependencies-without-changing-the-public-api)

Please check and see if you agree/disagree.

Reply:
Good point, I agree with you!